src/subcommand.rs

```diff
@@ -0,0 +1,121 @@
+//! A cage subcommand
+
+use std::str::FromStr;
+
+/// A cage subcommand
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
+pub enum Subcommand {
+
+    /// The `cage build` subcommand
+    Build,
```

Comment:
This is a reasonable way to approach this problem, but I think it might be better to define something like:

```rust
pub trait Subcommand {
    fn include_build_in_docker_compose(&self) -> bool {
        false
    }
    fn run(...) -> Result<()>;
}
```

...and port all of the subcommands in `src/cmd` to implement this interface. But this is a bigger change, and we don't have to do it right now.

Reply:
I like that idea, it might help shorten some of the big `match`s in `main.rs`.